docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -33,11 +33,11 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/
     - pattern: /docs/grafana-cloud/
       destination: docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/configure-tempo-data-source/
-  provisioning-data-sources:
+  explore-profiles:
```

Comment:
Thank you for using the refs like this. I did a double check since this change shows that you're deleting the provisioning-data-sources ref, which is a duplicate of one on line 21. Adding this note in case anyone else catches this.